caf-audit-service-container/pom.xml

```
@@ -275,7 +275,7 @@
                             <alias>keystore</alias>
                             <name>${project.artifactId}-keystore:${project.version}</name>
                             <build>
-                                <from>${dockerHubPublic}/cafapi/opensuse-jre11:3</from>
+                                <from>${dockerHubPublic}/cafapi/prereleases:opensuse-jre11-3.6.0-SNAPSHOT</from>
```

```suggestion
                                <from>${dockerHubPublic}/cafapi/opensuse-jre11:3</from>
```
Don't worry about updating the keystore images used in tests.